### DISCUSSION TITLE
Why does archive not displaying status?

### DISCUSSION THREAD

**Discussion by acceleratxr** [ericaltendorf/plotman#329] [Q&A] [answered]
Hi all, new to plotman but not chia. I've tried to set up plotman so that it will rsync all my finished plots to a remote server. On the remote server I have set up my rsyncd.conf as follows.

```
lock file = /var/run/rsync.lock
log file = /var/log/rsyncd.log
pid file = /var/run/rsync.pid

[chia]
        path = /chia
        comment = Chia Storage
        uid = ubuntu
        gid = ubuntu
        read only = no
        list = yes
        hosts allow = 10.0.0.0/255.255.0.0
```

I have configured plotman accordingly.

```
rsyncd_module: chia
rsyncd_path: /chia/plots
rsyncd_bwlimit: 100000
rsyncd_host: 10.0.0.40
rsyncd_user: ubuntu
```

I can confirm rsync is set up correctly and working as I can successfully send a test file via the command.

`rsync test.txt rsync://ubuntu@10.0.0.40/chia/plots`

But for some reason I don't see any status info in plotman interactive or using any of the other CLI commands.

![image](https://user-images.githubusercontent.com/43181229/117756203-98d0c780-b1d2-11eb-9fcb-25c56cf1eead.png)

I've also verified that I can successfully run the command...

`ssh ubuntu@10.0.0.40 df -aBK | grep /chia/plots`

with the output...

```
/dev/mapper/chia-plots              27234884536K 22744983500K 31226468165K 88% /chia/plots
```

Can anyone explain why this doesn't appear to work?

**Comment by acceleratxr** [comment, marked as answer]
OK I figured it out. I need to change the rsync path to match exactly what was in rsyncd.conf.

So I now have.

```
rsyncd_module: chia
rsyncd_path: /chia
...
```

That seems to have fixed it.

![image](https://user-images.githubusercontent.com/43181229/117757282-a8e9a680-b1d4-11eb-8dbd-bdf9043f8a16.png)

**Comment by mitchhuey** [comment]
It's actually cosmetic and a result of the ssh call here-  https://github.com/ericaltendorf/plotman/blob/5788e7fa2df2186ccb94db2d388766f9d52116a4/src/plotman/archive.py#L52

...and the trailing "/"
